packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -357,7 +374,7 @@ export async function sideloadAddIn(
   }
 }
 
-async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document?: string) {
+async function launchDesktopApp(app: OfficeApp, manifestPath: string, manifest: ManifestInfo, document?: string) {
```

Comment:
Instead of adding another argument to this method, we should add a manifestType and/or path property to ManifestInfo.  We shouldn't have to keep both pieces of information everywhere when keeping it together makes more sense.

Reply:
Ok, will change.